commands/global/new.js

```diff
@@ -256,4 +269,75 @@ module.exports.handler = async function handler(options) {
   );
   fs.outputFileSync(projectName + '/.gitignore', '/node_modules\n/.eslintcache');
   fs.ensureFileSync(projectName + '/transforms/.gitkeep');
+
+  // If import options [ url && codemod] are present
+  if (url && codemodName) {
+    let regex = /https:\/\/astexplorer\.net\/#\/gist\/(\w+)\/(\w+)/;
+    let matches = regex.exec(url);
+
+    let [, gist_id] = matches;
+
+    require('dotenv').config();
+    const Octokit = require('@octokit/rest');
+    const octokit = new Octokit({ auth: process.env.CODEMOD_CLI_API_KEY });
+    let codemodDir = `${process.cwd()}/${projectName}/transforms/${codemodName}`;
```

Comment:
Do we need this? Can we download the actual file directly (without needing an API key)?

I'm thinking like, from:

```
https://gist.githubusercontent.com/astexplorer/6f3bf899542dbe21e0474a2a74b2ffc9/raw/c23b658b268c2e5b644fed7f1a360433748e7f71/transform.js
```

Reply:
I think yes, but in case of export  we need an api to create gists, so I thought we can use this. What do you think?